docker/README.md

```diff
@@ -0,0 +1,30 @@
+# Ed-Fi-Analytics-Middle-Tier docker local configuration
+
+This docker container satisfies the user story:
+> As an AMT developer, I want to run integration tests using PostgreSQL and SQL
+> Server in Docker containers on my localhost.
+
+- Rename .env.example file to .env to be considered by docker compose
+- To start the containers run in terminal with administrative priviledges:
+```bash
+ docker compose up
```

Comment:
I think most people would want to start the containers in the background so that they can continue using their terminal.

```suggestion
 docker compose up -d
```